### PR TITLE
Feature/CVSB-10175 psv view

### PR DESCRIPTION
**Pull request by rspasici**

### CODE REVIEW

File: src/app/technical-record/body/__snapshots__/body.component.spec.ts.snap [hunk 1]
```diff
@@ -7,42 +7,13 @@ exports[`BodyComponent should create view only with populated data 1`] = `
...
-        class="govuk-summary-list__value"
-      />
-    </div>
+    
+    
+    
+    
+    
+    
+
```
Comment (alleypa) [inline src/app/technical-record/body/__snapshots__/body.component.spec.ts.snap:16]: why are we removing all the snapshots

File: src/app/technical-record/brakes/brakes.component.html [hunk 2]
```diff
@@ -19,3 +19,7 @@
       <vtm-axle-brakes [axle]="axle" [vehicleType]="currentVehicleType"></vtm-axle-brakes>
   </ng-container>
 </ng-container>
+
+<ng-container *ngIf="currentVehicleType === 'psv'">
```
Comment (alleypa) [inline src/app/technical-record/brakes/brakes.component.html:23]: Do we have test to cover when the vehicle type is not PSV from the perspective of the parent i.e. `brakes`

File: src/app/technical-record/dda/dda.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  selector: 'vtm-dda',
+  templateUrl: './dda.component.html',
+  styleUrls: ['../../app.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DdaComponent implements OnInit {
+
+  @Input() ddaDetails: DDA;
+
+  constructor(public techRecHelpers: TechRecordHelpersService) { }
```
Comment (alleypa) [inline src/app/technical-record/dda/dda.component.ts:15]: please dont use TechRecordHelpersService. The service has no value

File: src/app/technical-record/notes/notes.component.ts [hunk 4]
```diff
@@ -8,12 +8,9 @@ import {TechRecordHelpersService} from '@app/technical-record/tech-record-helper
...
 })
 export class NotesComponent implements OnInit {
-
   @Input() activeRecord: any;
 
-  constructor(public techRecHelpers: TechRecordHelpersService) { }
-
-  ngOnInit() {
-  }
+  constructor(public techRecHelpers: TechRecordHelpersService) {}
```
Comment (alleypa) [inline src/app/technical-record/notes/notes.component.ts:13]: same here

File: src/app/technical-record/notes/notes.component.ts [hunk 5]
```diff
@@ -8,12 +8,9 @@ import {TechRecordHelpersService} from '@app/technical-record/tech-record-helper
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class NotesComponent implements OnInit {
-
   @Input() activeRecord: any;
```
Comment (alleypa) [inline src/app/technical-record/notes/notes.component.ts:11]: use of `any`

File: src/app/technical-record/tyres/__snapshots__/tyres.component.spec.ts.snap [hunk 6]
```diff
@@ -10,33 +10,8 @@ exports[`TyresComponent should create view only with populated data 1`] = `
...
-        />
-        <th
-          class="govuk-table__cell"
-        />
-        <th
-          class="govuk-table__cell"
-        />
-      </tr>
+      
+
```
Comment (alleypa) [inline src/app/technical-record/tyres/__snapshots__/tyres.component.spec.ts.snap:14]: another snapshot getting lost

File: src/app/technical-record/vehicle-summary/vehicle-summary.component.html [hunk 7]
```diff
@@ -142,7 +141,7 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'hgv'">
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'psv'">
```
Comment (alleypa) [inline src/app/technical-record/vehicle-summary/vehicle-summary.component.html:144]: do we have another commit breaks up this file based on vehicleType

File: src/app/utils/testing.utils.ts [hunk 8]
```diff
@@ -79,6 +82,25 @@ function mockApplicantDetails(args?: Partial<ApplicantDetails>): ApplicantDetail
   return { ...mock, ...args };
 }
 
+function mockDDA(args?: Partial<DDA>): DDA {
+  const mock: DDA = {
```
Comment (alleypa) [inline src/app/utils/testing.utils.ts:86]: Reduce this to some essential keys and pass the remaining fields if relevant to the test via the `it` test

File: src/app/utils/testing.utils.ts [hunk 9]
```diff
@@ -336,5 +358,30 @@ function mockBodyType(args?: Partial<BodyType>): BodyType {
...
+
+  return { ...mock, ...args };
+}
+
+function mockPSVBrakes(args?: Partial<Brakes>): Brakes {
+  const mock: Brakes = {
+    brakeCodeOriginal: '123',
+    brakeCode: '145202',
+    retarderBrakeOne: 'electric',
+    brakeForceWheelsNotLocked: {
```
Comment (alleypa) [inline src/app/utils/testing.utils.ts:370]: this should have its own mock method

File: src/app/utils/testing.utils.ts [hunk 10]
```diff
@@ -336,5 +358,30 @@ function mockBodyType(args?: Partial<BodyType>): BodyType {
...
+      parkingBrakeForceA: 2322,
+      serviceBrakeForceA: 6531,
+      secondaryBrakeForceA: 3265
+    },
+    dataTrBrakeTwo: 'None',
+    retarderBrakeTwo: 'exhaust',
+    dataTrBrakeOne: 'None',
+    dataTrBrakeThree: 'None',
+    brakeForceWheelsUpToHalfLocked: {
```
Comment (alleypa) [inline src/app/utils/testing.utils.ts:379]: also this. see similar break ups in this file

File: src/app/utils/testing.utils.ts [hunk 11]
```diff
@@ -336,5 +358,30 @@ function mockBodyType(args?: Partial<BodyType>): BodyType {
   const mock: BodyType = {
     description: 'the first letter should be capital'
   } as BodyType;
+
+  return { ...mock, ...args };
+}
+
+function mockPSVBrakes(args?: Partial<Brakes>): Brakes {
+  const mock: Brakes = {
```
Comment (alleypa) [inline src/app/utils/testing.utils.ts:366]: Also limit the brake keys to essential fields capture on the screen. We should split the keys between this file and the spec under test

File: src/app/technical-record/weights/weights.component.html [hunk 12]
```diff
@@ -3,10 +3,13 @@
   <tr class="govuk-table__row">
     <th scope="col" class="govuk-table__header"></th>
     <th scope="col" class="govuk-table__header">GB weight (kg)</th>
-    <th scope="col" class="govuk-table__header">EEC weight (kg)</th>
+    <th scope="col" class="govuk-table__header" *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'">EEC weight (kg)</th>
```
Comment (alleypa) [inline src/app/technical-record/weights/weights.component.html:6]: let evaluate this logic `activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'"`. Anticipating complexity

File: src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+})
+export class PsvBrakesComponent implements OnInit {
+
+  @Input() brakes: Brakes;
+  
+  constructor() { }
+
+  ngOnInit() {
+    console.log(this.brakes);
+  }
```
Comment (alleypa) [inline src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts:17]: console.log

File: src/app/technical-record/body/body.component.html [hunk 14]
```diff
@@ -17,6 +17,57 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'psv'">
```
Comment (alleypa) [inline src/app/technical-record/body/body.component.html:21]: update all to use the `===`

File: src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts (hunk 13 above)
Comment (JonLazariniDD) [reply src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts:17]: +1

File: src/app/technical-record/body/body.component.html (hunk 14 above)
Comment (JonLazariniDD) [reply src/app/technical-record/body/body.component.html:21]: let's break body into children components that render the fields needed per vehicle types like it is done brakes.component (that is what we want)

We want the following (as folder structure):
```
/body
  body.component.html // will load body-psv fields in here
   ...
  /body-psv
    body-psv.component.html //only contains the body fields for psv 
```

```
// body-psv.component should look like what we have in here
// if we refer to the all structure mapping it has model literal, chassis make, chassis model fields
<div>
   <dt class="govuk-summary-list__key">
      Make
    </dt>
    <dd id="test-bodyMake" class="govuk-summary-list__value">
      {{ activeRecord.bodyMake | DefaultNullOrEmpty }}
    </dd>
    <dd class="govuk-summary-list__value"></dd>
     ..... model literal fields, etc...
</div>
```

Then body.omponent.html loads the common body fields + fields conditionally to specific vehicle types in our case psv:

```
// body.component.html
<div>
  ...field for model here
  ...field for make
  ...body type field
  ...function field
  ...conversion reference number 
  <ng-container *ngIf="currentVehicleType === 'psv'"> // this only contains fields for psv (model literal, chassis make, chassis model)
     <vtm-body-psv  [body]="body" />
  </ng-container>
</div>

```

File: src/app/technical-record/vehicle-summary/vehicle-summary.component.html [hunk 15]
```diff
@@ -223,6 +222,67 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
```
Comment (JonLazariniDD) [inline src/app/technical-record/vehicle-summary/vehicle-summary.component.html:226]: let's please do the same exercise as brakes and body for vehicle-summary please (see the common fields below in the screenshot).
Then psv vehicle summary should show these additional fields: speed limiter exempt, tacho, euro, fuel, number of wheels driven, emission, seat lower decks, seat upper decks, standing capacity, vehicle size, number of seat belts, seat belt installations


image

File: src/app/technical-record/weights/weights.component.html [hunk 16]
```diff
@@ -3,10 +3,13 @@
...
     <th scope="col" class="govuk-table__header">Design weight (kg)</th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
+    <td scope="col" class="govuk-table__header" *ngIf="activeRecord.vehicleType === 'psv'">Kerb weight (kg)</td>
```
Comment (JonLazariniDD) [inline src/app/technical-record/weights/weights.component.html:8]: We can also probably do the same exercise (body and brakes) here again if we have table format,
vehicle-weight, gross-vehicle-weight, design, axle weight, 
GB, design (might be dependent on axle number field so we have to check the behaviour of what is happening when we add axles, whether it populates a section with GB, Design (and kerb laden for PSV).
We could could be showing the common table head and cells elements
then show the ones for psv
axle number field might be its own separate module as well as it populate the form the other axle fields i believe.

File: src/app/utils/testing.utils.ts (hunk 9 above)
Comment (rspasici) [reply src/app/utils/testing.utils.ts:370]: it's part of the same accordion and everything. there can't be used outside of that component. shall i still split those ?

File: src/app/technical-record/body/body.component.html [hunk 17]
```diff
@@ -17,16 +24,22 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <ng-container *ngIf="currentVehicleType === 'psv'">
+    <vtm-psv-body [activeRecord]="activeRecord"></vtm-psv-body>
```
Comment (alleypa) [inline src/app/technical-record/body/body.component.html:29]: please adopt the convention of `vtm-body-psv` `vtm-body-hgv` for example. This will affects the files and selectors

File: src/app/technical-record/brakes/psv-brakes/psv-brakes.component.spec.ts [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PsvBrakesComponent);
+    component = fixture.componentInstance;
+    component.brakes = TESTING_UTILS.mockPSVBrakes();
+    component.brakes.brakeForceWheelsNotLocked = TESTING_UTILS.mockBrakeForceWheelsNotLocked();
+    component.brakes.brakeForceWheelsUpToHalfLocked = TESTING_UTILS.mockBrakeForceWheelsUpToHalfLocked();
```
Comment (alleypa) [inline src/app/technical-record/brakes/psv-brakes/psv-brakes.component.spec.ts:25]: TESTING_UTILS.mockPSVBrakes({
  brakeForceWheelsNotLocked: TESTING_UTILS.mockBrakeForceWheelsNotLocked(),
  brakeForceWheelsUpToHalfLocked : TESTING_UTILS.mockBrakeForceWheelsUpToHalfLocked()
})

`brakes` is the exposed variable

File: src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts [hunk 19]
```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Brakes } from '@app/models/tech-record.model';
+
+@Component({
+  selector: 'vtm-psv-brakes',
+  templateUrl: './psv-brakes.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PsvBrakesComponent implements OnInit {
```
Comment (alleypa) [inline src/app/technical-record/brakes/psv-brakes/psv-brakes.component.ts:9]: BrakesPsvComponent. Please follow this convention

File: src/app/technical-record/vehicle-summary/__snapshots__/vehicle-summary.component.spec.ts.snap [hunk 20]
```diff
@@ -5,335 +5,6 @@ exports[`VehicleSummaryComponent should create view only with populated data 1`]
...
-        id="test-variantVersionNumber"
-      >
-         - 
-      </dd>
-      <dd
-        class="govuk-summary-list__value"
-      />
-    </div>
-  </dl>
+
```
Comment (alleypa) [inline src/app/technical-record/vehicle-summary/__snapshots__/vehicle-summary.component.spec.ts.snap:8]: Entire snapshot is lost here, why?

File: src/app/technical-record/vehicle-summary/psv-vehicle-summary/psv-vehicle-summary.component.html [hunk 21]
```diff
@@ -0,0 +1,314 @@
+<dl class="technical-record-sub-details govuk-summary-list">
```
Comment (alleypa) [inline src/app/technical-record/vehicle-summary/psv-vehicle-summary/psv-vehicle-summary.component.html:1]: file naming should be vehicle-summary-psv

File: src/app/technical-record/weights/weights.component.ts [hunk 22]
```diff
@@ -9,12 +9,9 @@ import { TechRecord } from '../../models/tech-record.model';
...
 })
 export class WeightsComponent implements OnInit {
-
   @Input() activeRecord: TechRecord;
 
-  constructor(public techRecHelpers: TechRecordHelpersService) { }
-
-  ngOnInit() {
-  }
+  constructor(public techRecHelpers: TechRecordHelpersService) {}
```
Comment (alleypa) [inline src/app/technical-record/weights/weights.component.ts:14]: please remove

File: src/app/technical-record/body/body.component.html [hunk 23]
```diff
@@ -1,5 +1,9 @@
 <dl class="technical-record-sub-details govuk-summary-list ">
```
Comment (alleypa) [inline src/app/technical-record/body/body.component.html:1]: please we need to remove this css class if not used `technical-record-sub-details`

File: src/app/technical-record/notes/notes.component.html [hunk 24]
```diff
@@ -1,7 +1,22 @@
 <dl class="technical-record-sub-details govuk-summary-list">
```
Comment (alleypa) [inline src/app/technical-record/notes/notes.component.html:1]: we need to remove this css `technical-record-sub-details`

File: src/app/technical-record/dda/dda.component.ts [hunk 25]
```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { DDA } from '@app/models/tech-record.model';
+
+@Component({
+  selector: 'vtm-dda',
+  templateUrl: './dda.component.html',
+  styleUrls: ['../../app.component.scss'],
```
Comment (alleypa) [inline src/app/technical-record/dda/dda.component.ts:7]: we need to remove this line

File: src/app/technical-record/vehicle-summary/vehicle-summary.component.html [hunk 26]
```diff
@@ -283,6 +304,36 @@ <h2 class="govuk-heading-m">Type Approval</h2>
...
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+    <dt class="govuk-summary-list__key">
+      COIF date
+    </dt>
+    <dd id="test-vehicleSummary-coifDate" class="govuk-summary-list__value">
+      {{ activeRecord.coifDate | DefaultNullOrEmpty }}
+    </dd>
```
Comment (alleypa) [inline src/app/technical-record/vehicle-summary/vehicle-summary.component.html:334]: should all these fields be inside`vtm-vehicle-summary-psv`

File: src/app/technical-record/dda/dda.component.ts (hunk 25 above)
Comment (JonLazariniDD) [reply src/app/technical-record/dda/dda.component.ts:7]: nice one for spotting it, I don't know why we load this file referenced in app, we agreed we will be using _global.scss (if and only if new component styles are required for components that do not exist) and no scoped css

File: src/app/technical-record/body/body-psv/body-psv.component.spec.ts [hunk 27]
```diff
@@ -0,0 +1,45 @@
...
+      modelLiteral: 'test',
+      chassisMake: 'volvo',
+      chassisModel: 't23'
+    });
+    
+    fixture.detectChanges();
+  });
+
+  it('should create view only with populated data', () => {
+    fixture.detectChanges();
```
Comment (JonLazariniDD) [inline src/app/technical-record/body/body-psv/body-psv.component.spec.ts:36]: Something minor:
Not sure we need this again, we are calling it in the beforeEach() block already and we are not manipulating the component again by adding/modifying data so we don't need to call it again I believe.

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 28]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Service</dt>
+    <dd id="psv-brakes-serviceBrakeForceA" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsNotLocked?.serviceBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Secondary</dt>
+    <dd id="psv-brakes-secondaryBrakeForceA" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:62]: needs to be prepended by test- if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 29]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Retarder brake one</dt>
+    <dd id="psv-brakes-retarderBrakeOne" class="govuk-summary-list__value">
+      {{ brakes?.retarderBrakeOne| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Retarder brake two</dt>
+    <dd id="psv-brakes-retarderBrakeTwo" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:44]: needs to be prepended by test- if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 30]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Parking brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeThree" class="govuk-summary-list__value">
+      {{ brakes?.dataTrBrakeThree| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Retarder brake one</dt>
+    <dd id="psv-brakes-retarderBrakeOne" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:36]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 31]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Secondary brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeTwo" class="govuk-summary-list__value">
+      {{ brakes?.dataTrBrakeTwo| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeThree" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:28]: needs to be prepended by test- if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 32]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Service brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeOne" class="govuk-summary-list__value">
+      {{ brakes?.dataTrBrakeOne| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Secondary brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeTwo" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:20]: needs to be prepended by test- if this is required by automation testers, please check with Dragos

File: src/app/technical-record/technical-record.component.html [hunk 33]
```diff
@@ -1,9 +1,7 @@
 <div class="govuk-width-container">
   <a class="govuk-back-link" vtmBackButton>Back</a>
   <main class="govuk-main-wrapper">
-    <div
-      *ngIf="activeRecord.vehicleType == 'hgv' || activeRecord.vehicleType == 'trl'; else ifPSV"
-    >
+    <div>
```
Comment (JonLazariniDD) [inline src/app/technical-record/technical-record.component.html:4]: Was it a missing <div>?
Not sure why we need to be having an empty div which looks like a wrapper here? it has nothing on it

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 34]
```diff
@@ -0,0 +1,101 @@
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Brake code</dt>
+    <dd id="psv-brakes-brakeCode" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:4]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 35]
```diff
@@ -0,0 +1,101 @@
...
+      {{  brakes?.brakeCode | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Service brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeOne" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:12]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 36]
```diff
@@ -0,0 +1,101 @@
...
+      {{ brakes?.retarderBrakeTwo| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <br>
+  <h2 class="govuk-heading-m">Brake force wheels not locked</h2>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Service</dt>
+    <dd id="psv-brakes-serviceBrakeForceA" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:54]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 37]
```diff
@@ -0,0 +1,101 @@
...
+    <dt class="govuk-summary-list__key">Secondary</dt>
+    <dd id="psv-brakes-secondaryBrakeForceA" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsNotLocked?.secondaryBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking</dt>
+    <dd id="psv-brakes-parkingBrakeForceA" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:70]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 38]
```diff
@@ -0,0 +1,101 @@
...
+      {{ brakes?.brakeForceWheelsNotLocked?.parkingBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <br>
+  <h2 class="govuk-heading-m">Brake force wheels up to half locked</h2>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Service</dt>
+    <dd id="psv-brakes-serviceBrakeForceB" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:80]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 39]
```diff
@@ -0,0 +1,101 @@
...
+      {{ brakes?.brakeForceWheelsUpToHalfLocked?.serviceBrakeForceB| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Secondary</dt>
+    <dd id="psv-brakes-secondaryBrakeForceB" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:88]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html [hunk 40]
```diff
@@ -0,0 +1,101 @@
...
+      {{ brakes?.brakeForceWheelsUpToHalfLocked?.secondaryBrakeForceB| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking</dt>
+    <dd id="psv-brakes-parkingBrakeForceB" class="govuk-summary-list__value">
```
Comment (JonLazariniDD) [inline src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html:96]: needs to be prepended by test-* if this is required by automation testers, please check with Dragos

File: src/app/technical-record/vehicle-summary/vehicle-summary.component.html (hunk 26 above)
Comment (rspasici) [reply src/app/technical-record/vehicle-summary/vehicle-summary.component.html:334]: they are unde type approval not the other fields

File: src/app/technical-record/body/body.component.html (hunk 17 above)
Comment (JonLazariniDD) [reply src/app/technical-record/body/body.component.html]: Resolving as this is changed
